pkg/minikube/node/start.go

```diff
@@ -156,11 +156,24 @@ func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 			return nil, errors.Wrap(err, "Updating node")
 		}
 
-		cpBs, err := cluster.Bootstrapper(starter.MachineAPI, viper.GetString(cmdcfg.Bootstrapper), *starter.Cfg, starter.Runner)
+		// Load the control plane host
+		cp, err := config.PrimaryControlPlane(starter.Cfg)
 		if err != nil {
-			return nil, errors.Wrap(err, "Getting bootstrapper")
+			return nil, errors.Wrap(err, "Getting primary control plane")
+		}
+		cpHost, err := machine.LoadHost(starter.MachineAPI, driver.MachineName(*starter.Cfg, cp))
```

Comment:
could mustload Package be used for this ?

Reply:
yep, good call. mustload.Running does all this.